### DISCUSSION TITLE
Using Auto Startup with Android "Device Management Application" permission

### DISCUSSION THREAD

**Discussion by AkarinLiu** [RikkaApps/Shizuku#340] [Ideas]
I have an idea to make the phone boot without rooting.
Use the built-in "Device Management Application" permission of the Android operating system to realize Shizuku boot automatically, so that there is no need to manually start Shizuku services after booting.

**Comment by RikkaW** [comment]
The permission of "Device Management Application" (device or profile owner in fact) is so limited that could not match the requirement for most apps using Shizuku.
